gulpfile.js

```diff
@@ -0,0 +1,18 @@
+var gulp = require('gulp');
```

Comment:
Agreed that browsersync is not necessary and can be removed. It was just from another project that I used as reference.